djangocms_content_expiry/cms_config.py

```diff
@@ -1,6 +1,14 @@
 from django.conf import settings
 
-from cms.app_base import CMSAppConfig
+from cms.app_base import CMSAppConfig, CMSAppExtension
+
+
+class ContentExpiryAppExtension(CMSAppExtension):
+    def __init__(self):
+        from .monkeypatch import admin as monkeypatch_admin
```

Comment:
Monkey patching is normally done in the django apps.py -> ready method. Is there a reason you're trying to do it here int he cms config?

Reply:
@Aiky30 I worked off the version-locking example which had the admin patch in cms_config. Since moved it to apps.py